traffic_ops/testing/api/v3/profile_parameters_test.go

```diff
@@ -75,6 +75,11 @@ func TestProfileParameters(t *testing.T) {
 					}},
 					Expectations: utils.CkRequest(utils.HasError(), utils.HasStatus(http.StatusBadRequest)),
 				},
+				"BAD REQUEST when EMPTY BODY PROVIDED": {
+					ClientSession: TOSession,
+					RequestBody:   []tc.ProfileParameter{{}},
+					Expectations:  utils.CkRequest(utils.HasError(), utils.HasStatus(http.StatusBadRequest)),
+				},
```

Comment:
This doesn't actually test the behavior being fixed in this PR. This a list of length 1 (not 0!) whose only element has the default/"zero" values for all of its properties. [The result when Go marshals that is, accordingly, an array with one object element](https://go.dev/play/p/UadnzuI9JY3).